sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -611,10 +621,17 @@ export class CryptographyClient {
    * The authentication credentials
    */
   protected readonly credential: ServiceClientCredentials | TokenCredential;
+
+  /**
+   * @internal
+   * @ignore
+   * A reference to the auto-generated KeyVault HTTP client.
+   */
   private readonly client: KeyVaultClient;
 
   /**
-   * If the key is a string, it's a URL, and we'll pass it to the service API directly.
+   * A reference to the key used for the cryptographic operations.
+   * It can be either a string with the URL of a KeyVault Key, or an already parsed {@link JsonWebKey}.
```

Comment:
```suggestion
   * Based on what was provided to the CryptographyClient constructor, it can be either a string with the URL of a KeyVault Key, or an already parsed {@link JsonWebKey}.
```